plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp

```diff
@@ -260,8 +265,14 @@ void USentrySubsystem::ClearBreadcrumbs()
 void USentrySubsystem::AddAttachment(USentryAttachment* Attachment)
 {
 	check(SubsystemNativeImpl);
+	check(Attachment)
 
-	if (!SubsystemNativeImpl || !SubsystemNativeImpl->IsEnabled())
+		if (!SubsystemNativeImpl || !SubsystemNativeImpl->IsEnabled())
```

Comment:
hmm, why did the bot format it like this? :)

https://github.com/getsentry/sentry-unreal/pull/1105/commits/31207b1264f2806bac1612b8cd5e6759c45690e7

Reply:
Ah, missing `;` in the above line :sweat_smile: